inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution.cpp

```diff
@@ -66,4 +69,18 @@ void ConvolutionLayerTest::SetUp() {
     ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(conv)};
     function = std::make_shared<ngraph::Function>(results, params, "convolution");
 }
+
+ngraph::PartialShape ConvolutionLayerTest::vec2partialshape(std::vector<std::vector<size_t>> inputShape) {
```

Comment:
Is not it better to use vector of pairs here? This way we force people to initialize ranges correctly (with two values only).

Reply:
std::pair is used instead of std::vector. Thanks for your comment.